sw/host/opentitantool/src/main.rs

```diff
@@ -28,11 +28,10 @@ enum RootCommandHierarchy {
     Gpio(command::gpio::GpioCommand),
     Emulator(command::emulator::EmuCommand),
 
+    Fpga(command::fpga::FpgaCommand),
     I2c(command::i2c::I2cCommand),
     Image(command::image::Image),
-    SetPll(command::set_pll::SetPll),
     UpdateUsrAccess(command::update_usr_access::UpdateUsrAccess),
```

Comment:
nit: would you mind moving also this under the new command and updating references?

Reply:
As I understand it, implementing the reset command is a higher priority, so I filed #14950 to track moving `update-usr-access` under the `fpga` command.